lib/diaspora_federation/entity.rb

```diff
@@ -149,6 +149,19 @@ def to_s
       "#{self.class.name.rpartition('::').last}#{":#{guid}" if respond_to?(:guid)}"
     end
 
+    # Might be used to modify entity JSON object just before serialization
+    # @return [Hash] Returns a hash that is equal by structure to the entity in JSON format
+    def to_json_hash
+      {}
+    end
+
+    # @return [String] Renders the entity to the JSON representation
+    delegate :to_json, to: :to_json_hash
```

Comment:
I just disabled the cop for lib/* (and updated rubocop) in the develop branch. So it shouldn't rant after your next rebase :)